cli/src/index.ts

```diff
@@ -45,17 +46,17 @@ program
   .usage('[paths...] [options]')
   .addOption(new Option('-r, --recursive', 'Recursive').env('IMMICH_RECURSIVE').default(false))
   .addOption(new Option('-i, --ignore <pattern>', 'Pattern to ignore').env('IMMICH_IGNORE_PATHS'))
-  .addOption(new Option('-h, --skip-hash', "Don't hash files before upload").env('IMMICH_SKIP_HASH').default(false))
+  .addOption(new Option('-s, --skip-hash', "Don't hash files before upload").env('IMMICH_SKIP_HASH').default(false))
   .addOption(new Option('-H, --include-hidden', 'Include hidden folders').env('IMMICH_INCLUDE_HIDDEN').default(false))
   .addOption(
     new Option('-a, --album', 'Automatically create albums based on folder name')
       .env('IMMICH_AUTO_CREATE_ALBUM')
       .default(false),
   )
   .addOption(
-    new Option('-A, --album-name <name>', 'Add all assets to specified album')
+    new Option('-A, --album-name <n>', 'Add all assets to specified album')
       .env('IMMICH_ALBUM_NAME')
-      .conflicts('album'),
+      .conflicts('album')
```

Comment:
Can you explain why you are making these changes? Are these necessary/required to implement a hash-cache? If not, can you put them in a separate PR with some explanation of why they're being changed?